wordpress-mobile/WordPress-Android
# [Reader Improvements] Post card minor tweaks

Fixes #19337

Increase font size for all post card texts.

Note: I did not add the fallback to use the Author avatar in the cards since that can create an inconsistency with the Post Details / Site header, that are set to show the avatar only when there's a Blog avatar available.

To test:
1. Install and log into Jetpack
2. Go to Reader
3. Enable `ReaderImprovementsFeatureConfig`
4. **Verify** the post cards have bigger text in all feeds

## Regression Notes
1. Potential unintended areas of impact
N/A

5. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

6. What automated tests I added (or what prevented me from doing so)
N/A 

PR submission checklist:


UI Changes testing checklist:
